src/Microsoft.AspNet.Razor/Generator/Compiler/CodeTree/Chunks/TagHelpers/TagHelperPrefixDirectiveChunk.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Razor.Generator.Compiler
+{
+    /// <summary>
+    /// A <see cref="Chunk"/> for the @tagHelperPrefix directive.
+    /// </summary>
+    public class TagHelperPrefixDirectiveChunk : Chunk
+    {
+        /// <summary>
+        /// Text used as the prefix for <see cref="TagHelpers.TagHelperDescriptor"/>s.
```

Comment:
nit: storing a value in a descriptor isn't a "use". suggest `Text used as a required prefix when matching HTML start and end tags in the Razor source to available tag helpers.`  (might work without "in the Razor source")

similar comments about the descriptor rather than actual use appear elsewhere in this PR.